### PR TITLE
fix: [sc-32636] Downloading an object that doesn't exist doesn't relay an error to user

### PR DESCRIPTION
**Pull request by Cwagne17**
Story details: https://app.shortcut.com/clarkcan/story/32636

While trying to test downloads and bundling I needed to build an object that has files in minio so I tried doing it locally but came across a bunch of issues trying to even create a learning object. This PR fixes a lot of the issues experienced such as overwriting author and such.

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -15,7 +15,6 @@ export const environment = {
   clarkReportsUrl: 'http://localhost:9001',
   s3Bucket: 'clark-dev-file-uploads',
   s3BucketRegion: 'us-east-1',
-  s3Endpoint: 'http://localhost:9000',
```
Comment (johngonz0) [inline src/environments/environment.ts:18]: why is this removed?

**Review by johngonz0 — COMMENTED** [review]
good stuff, just a question

File: src/app/admin/pages/users/users.component.ts [hunk 2]
```diff
@@ -98,7 +98,6 @@ export class UsersComponent implements AfterViewInit {
         this.users.splice(0, 0, user);
       }).catch(error => {
         this.toaster.error('Error!', 'Could not add reviewer. Please try again later');
-        console.log(error);
```
Comment (pzalep1) [inline src/app/admin/pages/users/users.component.ts:101]: image

File: src/app/onion/learning-object-builder/builder-store.service.ts [hunk 3]
```diff
@@ -840,13 +837,11 @@ export class BuilderStore {
    * @memberof BuilderStore
    */
   private removeFiles(fileIds: string[]) {
-    console.log("remove files", fileIds);
```
Comment (pzalep1) [inline src/app/onion/learning-object-builder/builder-store.service.ts:843]: image

File: src/app/onion/learning-object-builder/components/content-upload/app/upload/upload.component.ts [hunk 4]
```diff
@@ -203,17 +205,24 @@ export class UploadComponent implements OnInit, AfterViewInit, OnDestroy {
         if (object) {
           this.learningObjectCuid = object.cuid;
           this.notes = object.materials.notes;
-          this.bucketUploadPath = `${object.author.username}/${object.id}`;
-          this.files$.next(object.materials.files);
+
+          // Temporarily store the learning obejct to test if it is being updated
+          this.learningObject = object;
```
Comment (pzalep1) [inline src/app/onion/learning-object-builder/components/content-upload/app/upload/upload.component.ts:210]: I bet if we looked at the amount of memory we use in this builder to store the learning object a bunch of times we would be really sad.

File: src/app/onion/shared/submit/submit.component.ts [hunk 5]
```diff
@@ -77,7 +77,6 @@ export class SubmitComponent implements OnInit {
   createChangelog() {
 
     if (this.changelog) {
-      console.log(this.changelog);
```
Comment (pzalep1) [inline src/app/onion/shared/submit/submit.component.ts:80]: image

**Review by pzalep1 — APPROVED** [review]

File: src/environments/environment.ts (hunk 1 above)
Comment (Cwagne17) [reply src/environments/environment.ts:18]: Hardcoded it so we don't have failing builds because other environment files have missing attribute.

File: src/app/onion/learning-object-builder/components/content-upload/app/upload/upload.component.ts (hunk 4 above)
Comment (Cwagne17) [reply src/app/onion/learning-object-builder/components/content-upload/app/upload/upload.component.ts:210]: ... yeahhhhhh

**Review by curlyLasagna — APPROVED** [review]

**Review by johngonz0 — APPROVED** [review]
